packages/hoppscotch-common/src/components/environments/my/index.vue

```diff
@@ -25,15 +32,31 @@
         />
       </div>
     </div>
+
     <EnvironmentsMyEnvironment
-      v-for="{ env, index } in alphabeticallySortedPersonalEnvironments"
+      v-for="{
+        env,
+        index,
+      } in searchedAndAlphabeticallySortedPersonalEnvironments"
       :key="`environment-${index}`"
       :environment-index="index"
       :environment="env"
       @edit-environment="editEnvironment(index)"
     />
     <HoppSmartPlaceholder
-      v-if="!alphabeticallySortedPersonalEnvironments.length"
+      v-if="
+        filterTexts &&
+        searchedAndAlphabeticallySortedPersonalEnvironments.length === 0
+      "
+      :alt="`${t('empty.search_environment')}`"
+      :text="`${t('empty.search_environment')} '${filterTexts}'`"
+    >
+      <template #icon>
+        <icon-lucide-search class="svg-icons opacity-75" />
+      </template>
+    </HoppSmartPlaceholder>
+    <HoppSmartPlaceholder
+      v-else-if="!searchedAndAlphabeticallySortedPersonalEnvironments.length"
```

Comment:
Can we conditionally render the markup block based on the filter text availability? The `icon` slot comes into action if filter text is present and the `body` slot otherwise.